### PR TITLE
81 carousel redux

### PR DESCRIPTION
**Pull request by OP-ReubenK**
Jackson has resolved file and image swapping on key press and it is ready to go to staging

### CODE REVIEW

File: react-arcade/src/App.js [hunk 1]
```diff
@@ -3,32 +3,40 @@ import React, { useEffect, useState } from "react";
...
-
+import Item from "./components/Game-Item";
+// import Image from ""
 const App = () => {
-  const [games, setGames] = useState([])
-  const [path, setPath] =  useState("")
-
+  const [games, setGames] = useState([]);
+  const [path, setPath] = useState("");
+  const [active, setActive] = useState({});
```
Comment (fuzzyers) [inline react-arcade/src/App.js:11]: Remove commented out code

File: react-arcade/src/App.js [hunk 2]
```diff
@@ -3,32 +3,40 @@ import React, { useEffect, useState } from "react";
...
+          ) : null}
+          {games.length > 0 ? (
+            <Carousel items={games} setActive={setActive} />
+          ) : (
             <div>Loading...</div>
-          }
-      </div>    
+          )}
+        </div>
+        {/* <div className="gameImg">erf</div> */}
```
Comment (fuzzyers) [inline react-arcade/src/App.js:35]: remove

File: react-arcade/src/components/Carousel.jsx [hunk 3]
```diff
@@ -1,68 +1,207 @@
-import React, { useState, useEffect, useCallback } from "react";
-import { TransitionGroup } from "react-transition-group";
-import "./App.scss";
-import { generateItems } from "../utils/Arcade-Controls";
+// import React, { useState, useEffect, useCallback } from "react";
```
Comment (fuzzyers) [inline react-arcade/src/components/Carousel.jsx:1]: remove

**Review by fuzzyers — APPROVED** [review]
Works well looks good plz remove commented code and console logs

**Review by ramandeepkau — APPROVED** [review]
Looks good, just an advise to clean up all the commented code lines

**Comment by OP-ReubenK** [comment]
those changes have been made

**Review by fuzzyers — APPROVED** [review]

**Pull request merged**